src/controller/CHIPDeviceController.cpp

```diff
@@ -792,6 +792,11 @@ void DeviceCommissioner::OnDiscoveredDeviceOverBleError(void * appState, CHIP_ER
     {
         self->ReleaseCommissioneeDevice(device);
         self->mRendezvousParametersForDeviceDiscoveredOverBle = RendezvousParameters();
+
+	if (self->mPairingDelegate != nullptr)
+	{
+            self->mPairingDelegate(err);
```

Comment:
This is not going to compile.  Was this tested?

Reply:
My fault, I tested locally, I committed the wrong branch code. I first withdraw the request.

Reply:
```
if (self->mPairingDelegate)
{
    self->mPairingDelegate->OnPairingComplete(err);
}
```